### PR TITLE
Strat

### PR DESCRIPTION
**Pull request by owain-Biddulph**

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ class LabelEncodingEnum(Enum):
 if MODEL == ModelEnum.BASIC_CNN:
     NB_EPOCHS = 20
 elif MODEL == ModelEnum.PIANOCKTAIL_GRU:
-    NB_EPOCHS = 300
+    NB_EPOCHS = 2000
```
Comment (quentinverlhac) [inline config.py]: I thought it was overfitting at 2000.
Shouldn't we keep this to 300 ?

File: split.py [hunk 2]
```diff
@@ -1,4 +1,7 @@
-from sklearn.model_selection import train_test_split
+from skmultilearn.model_selection.iterative_stratification import iterative_train_test_split
```
Comment (quentinverlhac) [inline split.py]: Is `skmultilearn` in `requirements.txt` ?

File: split.py [hunk 3]
```diff
@@ -9,15 +12,21 @@
 # Load processed data
 all_spectro = utils.load_dump(config.EMOTIFY_SPECTROGRAM_DUMP_PATH)
 
-labels = utils.load_labels(config.EMOTIFY_LABELS_DUMP_PATH)
```
Comment (quentinverlhac) [inline split.py]: If you don't need `utils.load_labels` anymore, could you remove the function ? Unless it is used somewhere else.

File: config.py (hunk 1 above)
Comment (owain-Biddulph) [reply config.py]: I don’t think it matters much for this merge, the best value will be determined when we have a validation set.

File: split.py (hunk 2 above)
Comment (owain-Biddulph) [reply split.py]: That is what was missing !

File: split.py (hunk 3 above)
Comment (owain-Biddulph) [reply split.py]: Don’t think it is really necessary to remove it right now, it could become useful.
